llvm/test/tools/llvm-cov/mcdc-macro.test

```diff
@@ -0,0 +1,92 @@
+// Test visualization of MC/DC constructs for branches in macro expansions.
+
+// RUN: llvm-profdata merge %S/Inputs/mcdc-macro.proftext -o %t.profdata
+// RUN: llvm-cov show --show-expansions --show-branches=count --show-mcdc %S/Inputs/mcdc-macro.o -instr-profile %t.profdata -path-equivalence=/tmp,%S/Inputs %S/Inputs/mcdc-macro.c | FileCheck %s
```

Comment:
I rely on a pre-built object file for now (as I did in a handful of other cases) because it allows for a targeted check.  I know these are tedious to update.  Based on suggestions from others in the last code reviews, I'm considering changing these tests to be runtime profile tests in the future, removing the need for the object files.  However, for now, I think it's useful for this test.